docs/DeveloperGuide.md

```diff
@@ -307,6 +307,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3a1. AddressBook shows an error message.
 
       Use case resumes at step 2.
+    
+**Use case: Add leaves to an employee**
+
+**MSS**
+
+1.  User requests to list employees
+2.  HeRon shows a list of employees
+3.  User requests to add a certain number of leaves to a specific employee in the list
+4.  HeRon adds the leaves to the employee
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+
```

Comment:
Perhaps can also consider the extension case where the given leaves is invalid, eg. Inputting an alphabet instead of a number, or if a negative number is given.

Reply:
Thanks for the suggestion! I have added an extension for this use case.